scrapper.py

```diff
@@ -1,7 +1,13 @@
 """
 Scrapper implementation
 """
-
+import json
+import re
+import requests
+import shutil
+from bs4 import BeautifulSoup
+from constants import ASSETS_PATH, CRAWLER_CONFIG_PATH
+from core_utils.article import Article
```

Comment:
it is better to separate imports of external and internal modules
moreover, they should be in alphabetical order

Reply:
when i put them alphabetically, code style says me to put them the way i did.

Reply:
code style tells you to place standard imports before external and external before local, and it is correct
What is also correct is that to make code more readable it is better to keep alphabetical order among standard imports, among external and among local
It does not contradict